### PR TITLE
feat(sessions): enable sorting on sessions table

### LINKED ISSUE #5278: [sessions][ui] enable sorting on the sessions table

### LINKED ISSUE #5279: [sessions][gql] enable server side sorting of sessions

### PR DESCRIPTION
**Pull request by RogerHYang**
resolves #5279 
resolves #5278

### CODE REVIEW

File: app/src/pages/project/SessionsTable.tsx [hunk 1]
```diff
@@ -93,7 +101,10 @@ export function SessionsTable(props: SessionsTableProps) {
       props.project
     );
   const tableData = useMemo(() => {
-    return data.sessions.edges.map(({ session }) => session);
+    return data.sessions.edges.map(({ session }) => ({
+      ...session,
+      tokenCountTotal: session.tokenUsage.total,
+    }));
   }, [data]);
```
Comment (axiomofjoy) [inline app/src/pages/project/SessionsTable.tsx:108]:
```suggestion
  }, [data.sessions]);
```

Comment (axiomofjoy) [reply app/src/pages/project/SessionsTable.tsx:108]: Not sure if this makes a difference, but might be worth a try.

**Review by axiomofjoy — APPROVED** [review]

File: app/src/pages/project/SessionsTable.tsx (hunk 1 above)
Comment (Parker-Stafford) [reply app/src/pages/project/SessionsTable.tsx:108]: it probably won't in this case but doesn't hurt

Comment (RogerHYang) [reply app/src/pages/project/SessionsTable.tsx:108]: ok. will change. thx

**Review by Parker-Stafford — APPROVED** [review]

File: app/src/pages/project/SessionsTable.tsx (hunk 1 above)
Comment (Parker-Stafford) [reply app/src/pages/project/SessionsTable.tsx]: either way is all good!

**Pull request merged**